src/components/xy_chart/series/line_series.js

```diff
@@ -67,15 +87,19 @@ EuiLine.propTypes = {
   /** Without a color set, a random EUI color palette color will be chosen */
   color: PropTypes.string,
   curve: PropTypes.string,
-  hasLineMarks: PropTypes.bool,
+  showLine: PropTypes.bool,
+  showLineMarks: PropTypes.bool,
+  lineSize: PropTypes.number,
```

Comment:
This was left out intentionally. removing options like this allow/force the implementation to have a more consistent look across Kibana.

Reply:
I know and I think it's the way to go but we have this options in Kibana. Maybe we can drop this, but I think we need to talk to @timroes before removing kibana features.